tracer/test/test-applications/integrations/Samples.Kafka/Samples.Kafka.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">1.4.3</ApiVersion>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">1.9.2</ApiVersion>
```

Comment:
Just a thought, we _could_ avoid changes in CI if we do this 🤔 Not sure which is preferable tbh

 ```suggestion
    <ApiVersion Condition="'$(ApiVersion)' == '' AND '$(OS)' == 'Windows_NT'">1.4.3</ApiVersion>
    <!-- OSX can't run on < 1.9.2 https://github.com/confluentinc/confluent-kafka-dotnet/issues/1755 -->
    <ApiVersion Condition="'$(ApiVersion)' == '' AND '$(OS)' != 'Windows_NT'">1.9.2</ApiVersion>
```

Reply:
I don't know either 🤷 
But your proposition looks ok, I guess we can do that too.